sections/memory_safety_in_rust.tex

```diff
@@ -35,6 +32,29 @@ \section{Memory Safety in Rust}
 
 \end{frame}
 
+
+%%%
+
+\begin{frame}[fragile]{Ergonomic scaffolding with \texttt{unwrap()}}
+Explicit error handling may seem like a huge barrier.
+
+Extracting values from \texttt{Option<T>}, \texttt{Result<T, E>} and may others fast with \texttt{unwrap()} and abort the program on failure:
```

Comment:
> Extracting values from \texttt{Option<T>}, \texttt{Result<T, E>} and may others fast with \texttt{unwrap()} and abort the program on failure:

This sentence does sound a bit strange, doesn't it? Maybe something like:
"One can use `unwrap` to extract values from `Option<T>` and `Result<T, E>` (and maybe other types as well). If extraction is not possible, it will panic"